tests/test_send_recv_two_workers.py

```diff
@@ -0,0 +1,296 @@
+import asyncio
+import multiprocessing
+import os
+import subprocess
+
+from distributed.comm.utils import from_frames, to_frames
+from distributed.protocol import to_serialize
+from distributed.utils import nbytes
+
+import cloudpickle
+import numpy as np
+import pytest
+import rmm
+import ucp
+from utils import more_than_two_gpus
+
+cmd = "nvidia-smi nvlink --setcontrol 0bz"  # Get output in bytes
+subprocess.check_call(cmd, shell=True)
+
+pynvml = pytest.importorskip("pynvml", reason="PYNVML not installed")
+ITERATIONS = 100
```

Comment:
@pentschev do you want to see this parameterized -- is 100 enough for all objects ?

Reply:
I guess parameterizing it would be better, but it's not absolutely necessary now I'd say. I tried running this for quite a while and this test seems to work, even increasing to 1000 iterations.